src/utils/OpenPepXLLF.cpp

```diff
@@ -189,6 +188,8 @@ class TOPPOpenPepXLLF :
 
     registerIntOption_("precursor:min_charge", "<num>", 3, "Minimum precursor charge to be considered.", false, true);
     registerIntOption_("precursor:max_charge", "<num>", 7, "Maximum precursor charge to be considered.", false, true);
+    registerIntList_("precursor:corrections", "<num>", ListUtils::create<int>("-2, -1, 0"), "Monoisotopic peak correction. Looks for possible monoisotopic precurser peaks at given number * (C13-C12) difference to the monoisotopic peak assumed by the instrument. These should be ordered from more extreme to less extreme corrections. Numbers later in the list will be preferred in case of ambiguities.", false, false);
```

Comment:
can you check if MSGFPlus / Comet / SimpleSearchEngine uses the same numbering scheme? e.g. if -1 or +1 is used?

Reply:
@timosachsenberg the MSGF+ documentation says this:
for `-ti -1,2` ( = n in the range from -1 to +2) it makes the corrections
`abs(exp-calc-n*1.00335Da)<20ppm`
so they start off from the experimental mass and subtract the given numbers.
Comet seems to do the same. I will adapt my scheme to theirs then.